modules/prebidmanagerAnalyticsAdapter.js

```diff
@@ -83,7 +83,7 @@ function collectUtmTagData() {
     if (newUtm === false) {
       utmTags.forEach(function (utmKey) {
         let itemValue = localStorage.getItem(`pm_${utmKey}`);
-        if (itemValue.length !== 0) {
+        if (itemValue !== null && typeof itemValue !== 'undefined' && itemValue.length !== 0) {
```

Comment:
very minor but simpler to just `if (itemValue && itemValue.length !== 0) {`

Reply:
edited this line. thank you.